instrumentation/dubbo-rpc/src/main/java/brave/dubbo/rpc/DubboParser.java

```diff
@@ -33,7 +33,7 @@
    */
   static @Nullable String method(Invocation invocation) {
     String methodName = invocation.getMethodName();
-    if ("$invoke".equals(methodName)) {
+    if ("$invoke" .equals(methodName)) {
```

Comment:
Revert

Reply:
I'll try to figure out why my IDE is now just doing that. I recently updated but didn't change the formatter..